### PR TITLE
Add react hooks, and react a11y linters

### PR DESCRIPTION
**Pull request by acourdavault**
## Description

Adding automated linting for a11y and react hooks (for now)

## Type of Change
- Non breaking change ('update', 'change')

## Note to Reviewers

Probably will require yarn clean && yarn up And restarting vscode to load the linters correctly

# examples of issues found
### Aria role incorrect
https://www.w3.org/TR/wai-aria-1.1/#role_definitions
image
Actual rendered dom
image
Fixed the role, detects another error
image
Fix:
```typescript
 <Typography
            role="heading"
            aria-level={2}
            variant="h2"
            className={classes.title}
            data-qa-title
          >
            Resize
          </Typography>
```

NOTE
Another way would be to fix and parse the level in Typography to inject aria-level property and aria heading in the future

Click event and no role button
image

### CODE REVIEW

**Comment by acourdavault** [comment]
Made a good number of fixes but we still have:
 - anchors that could be changed to buttons
 - img without alt text
 - a placeholder text missing
 - tabIndex issues

See https://jira.linode.com/browse/M3-3470

File: packages/manager/src/features/Support/Hively.tsx [hunk 1]
```diff
@@ -103,6 +104,7 @@ export const Hively: React.FC<Props> = props => {
             src={
               'https://secure.teamhively.com/system/smileys/icons/000/000/542/px_25/icon_indifferent.png'
             }
+            alt="Yellow face with an indifferent expression."
```
Comment (martinmckenna) [inline packages/manager/src/features/Support/Hively.tsx]: lmfao

Comment (WilkinsKa1) [reply packages/manager/src/features/Support/Hively.tsx]: 😬 fixed that!

**Comment by acourdavault** [comment]
Seems to be good, now.

File: packages/manager/src/components/EnhancedSelect/__mocks__/Select.tsx [hunk 2]
```diff
@@ -24,9 +24,13 @@ export default ({ options, value, onChange, errorText }: any) => {
   const _options = groupsToItems(options);
   return (
     <>
-      <select data-testid="select" value={value || ''} onChange={handleChange}>
+      <select data-testid="select" value={value || ''} onBlur={handleChange}>
```
Comment (martinmckenna) [inline packages/manager/src/components/EnhancedSelect/__mocks__/Select.tsx]: this line seems like a change in functionality - was the linter complaining about this being an `onChange` handler?

**Comment by acourdavault** [comment]
Made some small changes to new Longview files which blocked the build/merge

**Pull request merged**